README.md

```diff
@@ -136,21 +135,21 @@ SHOW TABLES IN hive.default;
 SELECT * FROM hive.default.iris;
 ```
 
-#### Option 3 [local presto-cli]
-`NB!` Check [required software section](#required-software) first.
+#### Option 3 [local Presto-cli]
+> :information_source: Check [required software section](#required-software) first.
 
-* in a terminal run a proxy and `presto-cli` session
+In a terminal run a proxy and `Presto-cli` session:
 ```text
 make presto-cli
 ```
 
```

Comment:
Might be worth to mention the problem with `docker --network=host` on MacOS, and cmd `make presto-cli` will work only on Linux machines
```suggestion
> ⚠️ : Command contains two docker containers with the flag `--network=host`, natively run on Linux. 
> ⚠️ : [MacOS*] Docker runs in a virtual machine on, in that case, you need to use local binary `consul` to install proxy and in another terminal local binary with `presto` cli to connect.
```

Reply:
Yes, thanks! I'll find an appropriate place to put this.